src/hooks/accounts/resolvers/useGetAccountsQueryResolver.tsx

```diff
@@ -1,49 +1,34 @@
-import { isArray } from 'lodash';
 import { useCallback } from 'react';
 import { useGetAccounts } from '../useGetAccounts';
-import { usePersistActiveAccount } from '../usePersistActiveAccount';
 import { useResolverToRef } from './useAccountsMutationResolvers';
+import { Account } from '../../../generated/graphql';
 
-export interface AccountsQueryResolverArgs {
-    isActive?: boolean
-}
-
-export const __typename = 'Account';
+// make sure the __typename is well typed
+export const __typename: Account['__typename'] = 'Account';
 
 export const useGetAccountsQueryResolver = () => {
-    const [persistedActiveAccount] = usePersistActiveAccount();
-    const getAccounts = useGetAccounts();
+  const getAccounts = useGetAccounts();
+
+  return {
+    accounts: useResolverToRef(
+      useCallback(
+        async (_obj) => {
+          const accounts = await getAccounts();
 
-    return useResolverToRef(
-        useCallback(async (
-            _obj,
-            args: AccountsQueryResolverArgs 
-        ) => {
-            const accounts = await getAccounts(
-                persistedActiveAccount?.id,
-                args?.isActive,
-            );
+          // if no results were found, return undefined/null
+          // this is useful when un-setting the active account
+          if (!accounts) {
+            return null;
+          }
 
-            // if no results were found, return undefined/null
-            // this is useful when un-setting the active account
-            if (!accounts) {
-                return null;
-            };
-    
-            return isArray(accounts)
-                ? accounts.map(account => ({
-                    ...account,
-                    __typename
-                }))
-                : ({
-                    // just a single account
-                    ...accounts,
-                    __typename
-                })
-        }, [
-            persistedActiveAccount,
-            getAccounts
-        ]),
-        'accounts'
-    )
-}
\ No newline at end of file
+          return accounts.map((account) => ({
+            ...account,
+            __typename,
```

Comment:
Do you  plan to extract this into the new resolver folder/file structure and use a `withTypename` helper as well?